fdbserver/DataDistribution.actor.cpp

```diff
@@ -1411,6 +1446,10 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 				ASSERT_WE_THINK(server_info.find(machine.second->serversOnMachine[0]->id) != server_info.end());
 				// Skip unhealthy machines
 				if (!isMachineHealthy(machine.second)) continue;
+				// Skip machine with incomplete locality
+				if (!isValidLocality(configuration.storagePolicy,
+				                     machine.second->serversOnMachine[0]->lastKnownInterface.locality))
+					continue;
```

Comment:
Small nit: I think braces around the body would make this cleaner given the condition is multi-line.